__tests__/__unit__/job/actions.unit.test.ts

```diff
@@ -342,10 +329,12 @@ describe("Jobs Actions Unit Tests - Function submitJcl", () => {
 
     it("Checking submit of active text editor content as JCL", async () => {
         createGlobalMocks();
-        blockMocks = createBlockMocks();
+        const blockMocks: any = createBlockMocks();
```

Comment:
Why any? Can't we just leave it without annotation? And in case it is _really_ needed and we could not properly type it then, please, use `unknown` instead of `any`.

Reply:
No, if you leave it as any the following never resolves to text:

````
        mocked(vscode.window.showQuickPick).mockReturnValueOnce(new Promise((resolve) => {
            resolve(blockMocks.datasetSessionNode.label);
        }));
````